app/controllers/teams_controller.rb

```diff
@@ -10,15 +10,15 @@ def create
     if @team.save
       redirect_to crime_path(@crime), notice: 'You applied to this job succesfully!'
     else
-      render :new
+      redirect_to crime_path(@crime), notice: 'You already applied to this job...'
```

Comment:
Can we change the notice to 'You've already applied for this job.' ? Just grammar there ^^